R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -2482,6 +2482,32 @@ test_that("union(), unionByName(), rbind(), except(), and intersect() on a DataF
   unlink(jsonPath2)
 })
 
+test_that("intersectAll() and exceptAll()", {
+  df1 <- createDataFrame(
+    list(list("a", 1),
+      list("a", 1),
+      list("a", 1),
+      list("a", 1),
+      list("b", 3),
+      list("c", 4)),
+      schema = c("a", "b"))
+  df2 <- createDataFrame(
+    list(list("a", 1), list("a", 1), list("b", 3)),
+    schema = c("a", "b"))
+  intersect_all_expected <- data.frame("a" = c("a", "a", "b"), "b" = c(1, 1, 3),
+                                       stringsAsFactors = FALSE)
+  except_all_expected <- data.frame("a" = c("a", "a", "c"), "b" = c(1, 1, 4),
+                                    stringsAsFactors = FALSE)
+  intersect_all_df <- arrange(intersectAll(df1, df2), df1$a)
```

Comment:
Strictly, the naming rule is `intersectAllDf` or `intersect.all.df` (see https://github.com/apache/spark/pull/17590#issuecomment-293732796)